scss/atoms/modals/_mixins.scss

```diff
@@ -0,0 +1,47 @@
+/*
+* ==========================================================================
+* File: _mixins.scss
+* For : Modals
+* ==========================================================================
+*/
+
+// Styles shared by modal's inner components
+$border-radius: map-get($modal-borders, radius);
+$padding: em(map-get($modal-padding, padding));
```

Comment:
Retrieved externally because apparently scss was compiling "0 15px" to just "15px". Let me know if I am doing something wrong.